_episodes/interviews/2023-03-31-josh-matthews.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Servo with Josh Matthews"
+date: 2023-03-31T15:31:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-03-31-josh-matthews.mp3
+duration: "51:53"
+length: "49807778"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Josh Matthews](https://www.linkedin.com/in/joshuadmatthews/), former Engineering Manager at Mozilla for [Sevo](https://servo.org/), a web browser engine written in Rust.
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Josh's history with Servo
+- [@3:54] - Fears around the future of Rust
+- [@6:52] - Chrome’s Javascript engine
+- [@10:54] - How to make web design more secure with Rust
+- [@19:50] - What Servo is working on with Rust technology
+- [@24:49] - Servo's edge of responsibility as browser engine
```

Comment:
I don't know what you mean by "edge" here?

Reply:
this is talking about how much work Servo should do compared to other pieces of the browser